sceptre/sageit/templates/admodeladexplorer-redirector.yaml

```diff
@@ -0,0 +1,150 @@
+  AcmCertificateArn:
+    Description: ARN of the ACM certificate to apply to the Cloudfront distribution
+    Type: String
+    AllowedPattern: "arn:aws:acm:.*"
+    ConstraintDescription: must be a valid certificate ARN that covers all the domains below (at least prod and staging)
+Conditions:
+  IsProdEnv: !Equals [prod, !Ref StackType]
+Resources:
+  WebsiteBucket:
+    Type: AWS::S3::Bucket
+    Properties:
+      AccessControl: PublicRead
+      BucketName: !Sub
+      - ${stackPrefix}.admodeladexplorer.org
+      - { stackPrefix: !Ref StackType }
+      WebsiteConfiguration:
+        IndexDocument: index.html
+        ErrorDocument: index.html
+  WebsiteBucketPolicy:
+    Type: AWS::S3::BucketPolicy
+    Properties:
+      PolicyDocument:
+        Id: MyPolicy
+        Version: 2012-10-17
+        Statement:
+          - Sid: PublicReadForGetBucketObjects
+            Effect: Allow
+            Principal: '*'
+            Action: 's3:GetObject'
+            Resource: !Join ['', ['arn:aws:s3:::', !Ref WebsiteBucket, /*]]
+      Bucket: !Ref WebsiteBucket
+  Cloudfront:
+    DependsOn: RedirectFct
+    Type: AWS::CloudFront::Distribution
+    Properties:
+      DistributionConfig:
+        Comment: Cloudfront Distribution pointing to S3 bucket
+        Origins:
+          - DomainName: !Sub
+            - ${stackPrefix}.admodeladexplorer.org
+            - {stackPrefix: !Ref StackType}
+            Id: S3Origin
+            CustomOriginConfig:
+              HTTPPort: 80
+              HTTPSPort: 443
+              OriginProtocolPolicy: http-only
+        Enabled: true
+        HttpVersion: 'http2'
+        DefaultRootObject: index.html
+        Aliases:
+          - !Sub
+            - ${stackPrefix}.admodelexplorer.org
+            - { stackPrefix: !Ref StackType }
+          - !Sub
+            - ${stackPrefix}.modeladexplorer.org
+            - { stackPrefix: !Ref StackType }
+          - !If
+            - IsProdEnv
+            - admodelexplorer.org
+            - !Ref AWS::NoValue
+          - !If
+            - IsProdEnv
+            - modeladexplorer.org
+            - !Ref AWS::NoValue
+        CustomErrorResponses:
+          - ErrorCachingMinTTL: 60
+            ErrorCode: 404
+            ResponseCode: 200
+            ResponsePagePath: '/index.html'
+          - ErrorCachingMinTTL: 60
+            ErrorCode: 403
+            ResponseCode: 200
+            ResponsePagePath: '/index.html'
+        DefaultCacheBehavior:
+          DefaultTTL: 3600
+          AllowedMethods:
+            - GET
+            - HEAD
+          Compress: true
+          TargetOriginId: S3Origin
+          ForwardedValues:
+            QueryString: true
+            Cookies:
+              Forward: none
+          FunctionAssociations:
+          -
+            EventType: viewer-request
+            FunctionARN: !Sub
+            - arn:aws:cloudfront::797640923903:function/${stackPrefix}-modeladexplorer-redirector-fct
```

Comment:
can we use a  ${AWS::AccountId} instead of hard coding the account id?

Reply:
Done